### PR TITLE
add example: fine tune

### PR DESCRIPTION
**Pull request by mehulgohil**
**Describe the change**
Added an example for usage of fine tune

**Describe your solution**
example in readme file

Issue: #315

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -611,6 +611,73 @@ if errors.As(err, &e) {
...
+		fmt.Printf("Upload JSONL file error: %v\n", err)
+		return
+	}
+
+	//create a fine tune job
+	//Streams events until the job is done (this often takes minutes, but can take hours if there are many jobs in the queue or your dataset is large)
+	//use below get method to know the status of your model
+	tune, err := client.CreateFineTune(ctx, openai.FineTuneRequest{
+		TrainingFile: file.ID,
+		Model:        "ada", //babbage,curie,davinci
```
Comment (vvatanabe) [inline README.md:652]: Please add  "or a fine-tuned model created after 2022-04-21.".

> You can select one of "ada", "babbage", "curie", "davinci", or a fine-tuned model created after 2022-04-21. 

https://platform.openai.com/docs/api-reference/fine-tunes/create#fine-tunes/create-model

File: README.md [hunk 2]
```diff
@@ -611,6 +611,73 @@ if errors.As(err, &e) {
...
+	client := openai.NewClient("your token")
+	ctx := context.Background()
+
+	//create a .jsonl file with your training data
+	//{"prompt": "<prompt text>", "completion": "<ideal generated text>"}
+	//{"prompt": "<prompt text>", "completion": "<ideal generated text>"}
+	//{"prompt": "<prompt text>", "completion": "<ideal generated text>"}
+
+	//you can use openai cli tool to validate the data
+	//For more info - https://platform.openai.com/docs/guides/fine-tuning
```
Comment (vvatanabe) [inline README.md:636]: It would be more readable if you put spaces between "//" and the text.

**Comment by mehulgohil** [comment]
@vvatanabe have resolved your review

**Review by vvatanabe — APPROVED** [review]

**Comment by vvatanabe** [comment]
@mehulgohil Thanks for helping!

**Pull request merged**